spec/javascripts/components/story/story_description_spec.js

```diff
@@ -29,7 +29,7 @@ describe('<StoryDescription />', function() {
         value={""}
       />
     );
-    expect(wrapper.find('.description')).toHaveText('Description');
+    expect(wrapper.find('.description').text()).toContain('Description');
```

Comment:
What was the need of this change?

Reply:
The matcher was not finding the text for some reason after the "transition" to md